cluster_traffic_capture/cluster_traffic_capture/gen_haproxy_cfg.py

```diff
@@ -31,6 +31,11 @@ def _gen_global_config() -> str:
     log 127.0.0.1:514 len 65535 local0 # syslogd, facility local0
     log stdout format raw local0 debug # stdout too
 
+    # Increase the buffer size to allow both logging and spoa to forward large requests
+    tune.bufsize 1048576
+    tune.h2.max-frame-size 1048576
+    maxconn 65536
```

Comment:
Can you please add a comment explaining how these numbers were chosen?  Per the comments in [1] and underlying documentation [2], they're impactful to system performance and hypothetically we should be decreasing maxconn in proportion to increase in buffer size.

[1] https://github.com/haproxytech/spoa-mirror/issues/17#issuecomment-683848851
[2] https://cbonte.github.io/haproxy-dconv/2.2/configuration.html#3.2-tune.bufsize